mdbx/mdbxgo.c

```diff
@@ -113,6 +113,6 @@ int mdbxgo_dcmp(MDBX_txn *txn, MDBX_dbi dbi, char *adata, size_t an, char *bdata
     return mdbx_dcmp(txn, dbi, &a, &b);
 }
 
-
-
-
```

Comment:
Please take a look if we really need this wrapper. Other wrappers was created to reduce amount of CGO allocations. But allocations amount here are not important and seems func doesn’t do anything. Can call mdbx’s func directly.

Reply:
I removed the wrapper. I am thinking if `MdbxEnvWarmup` has to be struct method?

Reply:
Yes. Seems method of Txn 
Can’t pass nil txn to this func?